src/test/regress/expected/multi_generate_ddl_commands.out

```diff
@@ -30,12 +30,11 @@ SELECT master_get_table_ddl_events('not_null_table');
 -- ensure tables not in search path are schema-prefixed
 CREATE SCHEMA not_in_path CREATE TABLE simple_table (id bigint);
 SELECT master_get_table_ddl_events('not_in_path.simple_table');
-                  master_get_table_ddl_events                   
-----------------------------------------------------------------
- CREATE SCHEMA IF NOT EXISTS not_in_path AUTHORIZATION postgres
```

Comment:
`master_get_table_ddl_events()` should create a `CREATE SCHEMA` query here. I will look further into this

Reply:
Not sure that's strictly necessary. What are the implications?

Reply:
After taking a better look at how we use `master_get_table_ddl_events()` I guess it's ok to remove `CREATE SCHEMA` queries from here.

If we do not have the schema queries listed here, this test case becomes obsolete. That's why I am removing it